src/quemb/kbe/lo.py

```diff
@@ -118,19 +118,15 @@ def localize(
                 S12, S2 = get_xovlp_k(self.cell, self.kpts, basis=valence_basis)
                 ciao_ = get_iao_k(Co, S12, self.S, S2=S2)
 
-                arrange_by_atom = True
                 # tmp - aos are not rearrange and so below is not necessary
-                if arrange_by_atom:
-                    nk, nao, nlo = ciao_.shape
-                    Ciao_ = numpy.zeros((nk, nao, nlo), dtype=numpy.complex128)
-                    for k in range(self.nkpt):
-                        aoind_by_atom = get_aoind_by_atom(self.cell)
-                        ctmp, iaoind_by_atom = reorder_by_atom_(
-                            ciao_[k], aoind_by_atom, self.S[k]
-                        )
-                        Ciao_[k] = ctmp
-                else:
-                    Ciao_ = ciao_.copy()
+                nk, nao, nlo = ciao_.shape
+                Ciao_ = numpy.zeros((nk, nao, nlo), dtype=numpy.complex128)
+                for k in range(self.nkpt):
+                    aoind_by_atom = get_aoind_by_atom(self.cell)
+                    ctmp, iaoind_by_atom = reorder_by_atom_(
+                        ciao_[k], aoind_by_atom, self.S[k]
+                    )
+                    Ciao_[k] = ctmp
```

Comment:
As the comment suggests, reordering might be unnecessary. Verify this when dealing with #60

Reply:
but for the moment I leave it it as is?

Reply:
yes yes! Sorry should've been clearer :)